### PR TITLE
Spelling: Password needs

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: lib/PasswordValidator.php [hunk 1]
```diff
@@ -125,7 +125,7 @@ protected function checkSpecialCharacters($password) {
 		if($enforceSpecialCharacters && ctype_alnum($password)) {
 			$message = 'Password need to contain at least one special character.';
```
Comment (nickvergessen) [inline lib/PasswordValidator.php]: missing here as well

File: lib/PasswordValidator.php [hunk 2]
```diff
@@ -107,7 +107,7 @@ protected function checkNumericCharacters($password) {
 			if (preg_match('/^(?=.*\d).+$/', $password) !== 1) {
 				$message = 'Password need to contain at least one numeric character';
```
Comment (nickvergessen) [inline lib/PasswordValidator.php]: missing here as well

File: lib/PasswordValidator.php [hunk 3]
```diff
@@ -88,7 +88,7 @@ protected function checkUpperLowerCase($password) {
 			if (preg_match('/^(?=.*[a-z])(?=.*[A-Z]).+$/', $password) !== 1) {
 				$message = 'Password need to contain at least one lower case character and one upper case character.';
```
Comment (nickvergessen) [inline lib/PasswordValidator.php]: missing here as well

File: lib/PasswordValidator.php [hunk 4]
```diff
@@ -70,7 +70,7 @@ protected function checkPasswordLength($password) {
 		if(strlen($password) < $minLength) {
 			$message = 'Password need to be at least ' . $minLength . ' characters long';
```
Comment (nickvergessen) [inline lib/PasswordValidator.php]: missing here as well

**Comment by comradekingu** [comment]
@nickvergessen I fixed.

**Pull request merged**